prismicio/slice-machine
# Generate Slice index before a rename to avoid error message in user app

Prevent user's (Next at least) project to display an error when renaming slice, by generating the index file before renaming the Slice folder.

To test:
- pull this branch
- in packages/slice-machine-ui, run `npm run build && npm run export-build` 
- go to `e2e-projects/next`
- run `npm run slicemachine` and `npm run dev`
- rename a slice and check the last opened console for errors
